tests/test_instantiate.py

```diff
@@ -29,7 +29,7 @@ def example_instance(tmpdir_factory):
                 yield instance_path
             finally:
                 try:
-                    pip.main(['uninstall', 'ipywidgettestwidgets', '-y'])
+                    pip.main(['uninstall', 'jupyter_widget_testwidgets', '-y'])
                 except Exception:
                     pass
```

Comment:
should the test fail here if unable to uninstall? i.e. if the hyphen replacement didn't work correctly

Reply:
Turns out other things will also fail if the hyphen replacement didn't work so failing here probably isn't necessary